integration/duckdb/CMakeLists.txt

```diff
@@ -66,17 +65,26 @@ if(LANCE_BUILD_PYTORCH)
       set(torch_SOURCE_DIR ${CMAKE_BINARY_DIR}/thirdparty/torch/)
     endif()
   else()
-    FetchContent_Declare(
-      Torch
-      URL https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.12.1%2Bcpu.zip
-    )
+    if(LANCE_BUILD_CUDA)
+      FetchContent_Declare(
+        Torch
+        URL https://download.pytorch.org/libtorch/cu117/libtorch-cxx11-abi-shared-with-deps-1.13.0%2Bcu117.zip
+      )
+      enable_language(CUDA)
+      find_package(CUDAToolkit)
+    else()
+      FetchContent_Declare(
+        Torch
+        URL https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.13.0%2Bcpu.zip
+      )
+    endif()
     list(APPEND available_contents Torch)
   endif()
 endif()
 
 FetchContent_MakeAvailable(${available_contents})
 
-set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
why downgrade CXX standard here?

Reply:
nvcc doesn't support C++20

Reply:
so if this is set to 20, cmake complains about not supporting CUDA20